bin/alert_on_low_github_actions_quota.py

```diff
@@ -0,0 +1,55 @@
+from services.github_service import GithubService
+from config.constants import ENTERPRISE, MINISTRY_OF_JUSTICE, SLACK_CHANNEL
+import os
+from services.slack_service import SlackService
+import datetime
+
+def reset_alerting_threshold_if_first_day_of_month(github_service):
+    base_alerting_threshold = 70
+
+    if datetime.date.today().day == 1:
+        github_service.modify_gha_minutes_quota_threshold(base_alerting_threshold)
+
+def calculate_percentage_used(total_minutes_used):
+    total_available = 50000
+
+    return (total_minutes_used / total_available) * 100
+
+def low_threshold_triggered_message(percentage_used):
+    return f"Warning:\n\n {round(100 - percentage_used, 1)}% of the Github Actions minutes quota remains."
+
+def calculate_total_minutes_used(organisations, github_service):
```

Comment:
The fact that you're passing the GitHub service indicates it exists in the wrong place. The same can be said for `reset_alerting_threshold...`

I would look at refactoring and making this a method for that class.